### PR TITLE
Fix #86: Executable war

### LINKED ISSUE #86: Executable war
Add embedded tomcat as the provided dependency (to be part of `WEB-INF/lib-provided` but not `lib`) and make it possible to run it via `java -jar`.

### PR DESCRIPTION
**Pull request by jandusil**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 spring.application.name=user-data-store
 
-spring.datasource.url=${USER_DATA_STORE_DATASOURCE_URL:jdbc:postgresql://host.docker.internal:5432/powerauth}
+spring.datasource.url=${USER_DATA_STORE_DATASOURCE_URL:jdbc:postgresql://localhost:5432/powerauth}
```
Comment (banterCZ) [inline src/main/resources/application.properties:3]: Is this change needed?

Comment (jandusil) [reply src/main/resources/application.properties:3]: @banterCZ Not in theory, just trying to set some "standard". In Dockerized projects like pac and pa-tests we do have 
`spring.datasource.url=jdbc:postgresql://localhost:5432/powerauth` here.

Comment (banterCZ) [reply src/main/resources/application.properties:3]: If so, it does not belong to this PR anyway.

Comment (jandusil) [reply src/main/resources/application.properties:3]: Arlight

**Review by banterCZ — APPROVED** [review]
Looks good.

**Review by banterCZ — CHANGES_REQUESTED** [review]
No

**Review by banterCZ — APPROVED** [review]
Perfect!
